src/pool/srv_pool.c

```diff
@@ -1474,6 +1529,7 @@ ds_pool_create_handler(crt_rpc_t *rpc)
 	/* duplicate the default properties, overwrite it with pool create
 	 * parameter and then write to pool meta data.
 	 */
+	lazy_init_default_pool_props();
```

Comment:
Also, do we free them?

Reply:
Good catch. I only saw them used in creation - however, I'm not sure if there's a potential future usage. I agree it would make more sense to alloc/free on module init/fini.